.changeset/pretty-drinks-itch.md

```diff
@@ -0,0 +1,6 @@
+---
+"nextjs-website": patch
```

Comment:
```suggestion
"nextjs-website": minor
```
is this a minor, or am I missing something?
I mean, we are adding something, we are not fixing something broken.

If you have doubts, take a look at [semver.org](https://semver.org/)